packages/next/server/lib/utils.ts

```diff
@@ -12,3 +14,21 @@ export function getNodeOptionsWithoutInspect() {
   const NODE_INSPECT_RE = /--inspect(-brk)?(=\S+)?( |$)/
   return (process.env.NODE_OPTIONS || '').replace(NODE_INSPECT_RE, '')
 }
+
+export function getPort(args: arg.Result<arg.Spec>): number {
+  if (process.env.__NEXT_FORCED_PORT) {
+    // we allow the server to use a random port while testing
```

Comment:
This was moved and centralized here as well. Depending on why this was added, it may no longer be necessary itself now that this bug is fixed.